census_api_function/census_api_function.R

```diff
@@ -0,0 +1,161 @@
+# function for pulling clean acs data: clean_acs
+# Author: David Whyman
+# Date: Wed Jul 03 11:30:00 2019
+# SET UP ==============================================
+
+pkgs <- c("tidyverse", "tidycensus","skimr")
+
+check <-
+  sapply(pkgs,
+         require,
+         warn.conflicts = TRUE,
+         character.only = TRUE)
+
+if (any(!check)) {
+  pkgs.missing <- pkgs[!check]
+  install.packages(pkgs.missing)
+  check <-
+    sapply(pkgs.missing,
+           require,
+           warn.conflicts = TRUE,
+           character.only = TRUE)
+}
+
+census_api_key("e5ae735f65e6b93f5c2f9bc16bea67b72fa202da")
```

Comment:
Treat api key as your password - so don't post it to a public repo! Instead, you can [install api keys ](https://walkerke.github.io/tidycensus/reference/census_api_key.html)to your local environment.